Automattic/wp-calypso
# Jetpack Manage: Creating a child component of SingleSellUpsellLightbox

Follow-on from https://github.com/Automattic/wp-calypso/pull/86178

## Proposed Changes

* This PR extends the above PR by converting some of the content within the SingleSiteUpsellComponent into a child component.
* This PR also makes sure the 'issue license' button redirects you straight to the add card page, instead of the issue licenses page.
* This PR also fixes a typo in the previous components name.

## Testing Instructions

* Testing instructions should be exactly the same as those listed here: https://github.com/Automattic/wp-calypso/pull/86178
* When testing the issue licenses button, you should now be directed straight to the 'add card' page, not the issue licenses page.

<img width="1067" alt="Screenshot 2024-01-12 at 19 40 40" src="https://github.com/Automattic/wp-calypso/assets/16754605/0d9cb782-ddd4-4450-a96f-c983c59027e7">


## Pre-merge Checklist
